src/javascripts/components/search.js

```diff
@@ -117,6 +121,18 @@ Search.prototype.init = function () {
     return
   }
 
+  // The Accessible Autocomplete only works in IE9+ so we can use newer JavaScript features here
+  // but need to check for browsers that do not have these features and force the fallback by returning early.
+  // http://responsivenews.co.uk/post/18948466399/cutting-the-mustard
```

Comment:
are we adding this in here because it's necessary or because we can?

Reply:
I guess I could in theory write all this code in a way that supports IE8, but since the autocomplete does not work in IE8, it seems pointless?